From b04b22a38053c86b1502ef9bf0cddb8dda4db93d Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Mon, 20 Jul 2009 16:21:06 +0300
Subject: [PATCH] camerabin: remove unused variable assignment

Result is not needed here, but variable is used later on. Also remove double new
lines in function.
---
 gst/camerabin/camerabinvideo.c |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/gst/camerabin/camerabinvideo.c b/gst/camerabin/camerabinvideo.c
index 40a2bfa..bc91524 100644
--- a/gst/camerabin/camerabinvideo.c
+++ b/gst/camerabin/camerabinvideo.c
@@ -553,7 +553,6 @@ gst_camerabin_video_create_elements (GstCameraBinVideo * vid)
   gst_ghost_pad_set_target (GST_GHOST_PAD (vid->sinkpad), vid_sinkpad);
   gst_object_unref (vid_sinkpad);
 
-
   /* Add queue element for video */
   vid->tee_video_srcpad = gst_element_get_request_pad (vid->tee, "src%d");
   if (!(vid->video_queue =
@@ -565,7 +564,6 @@ gst_camerabin_video_create_elements (GstCameraBinVideo * vid)
   gst_pad_add_buffer_probe (vid->tee_video_srcpad,
       G_CALLBACK (camerabin_video_pad_tee_src0_have_buffer), vid);
 
-
 #ifdef USE_TIMEOVERLAY
   /* Add timeoverlay element to visualize elapsed time for debugging */
   if (!(gst_camerabin_create_and_add_element (vidbin, "timeoverlay"))) {
@@ -615,10 +613,9 @@ gst_camerabin_video_create_elements (GstCameraBinVideo * vid)
   }
 
   /* Add queue element for audio */
-  if (!(queue = gst_camerabin_create_and_add_element (vidbin, "queue"))) {
+  if (!(gst_camerabin_create_and_add_element (vidbin, "queue"))) {
     goto error;
   }
-  queue = NULL;
 
   /* Add optional audio conversion and volume elements and
      raise no errors if adding them fails */
